src/components/Groups/CreateGroupButton.tsx

```diff
@@ -2,21 +2,28 @@ import React, { Component } from 'react';
 import { compose } from 'react-apollo';
 import { withNavigation } from 'react-navigation';
 import { Button, ButtonText } from '../basic';
+import { StyleSheet } from 'react-native';
 
-export class CreateGroupButton extends Component {     
+export class CreateGroupButton extends Component {
     handlePress = () => {
         this.props.navigation.navigate('CreateGroup');
     }
 
     render() {
         return (
-            <Button onPress={this.handlePress}>
+            <Button onPress={this.handlePress} style={styles.createGroupb}>
                 <ButtonText>Create Group</ButtonText>
             </Button>
         );
     }
 }
 
+const styles = StyleSheet.create({
```

Comment:
Lets try to get these style sheets into styled-components. Look at the other components for how to do this.